gsa/src/web/pages/policies/component.js

```diff
@@ -766,7 +766,16 @@ class PolicyComponent extends React.Component {
                   onEditNvtDetailsClick={this.openEditNvtDetailsDialog}
                   onSave={d => {
                     this.handleInteraction();
-                    return save(d).then(() => this.closeEditPolicyDialog());
+                    // eslint-disable-next-line no-shadow
+                    const {name, comment, id} = d;
```

Comment:
Since we need multiple lines for this we probably should not put it all in here but add a handleSavePolicy function instead.